gds/include/cufile_stub.h

```diff
@@ -16,7 +16,12 @@
 #ifndef CUCIM_CUFILE_STUB_H
 #define CUCIM_CUFILE_STUB_H
 
-#include "cufile.h"
+// Try to include the real cufile.h, fall back to minimal types if not available
+#if __has_include(<cufile.h>)
+    #include <cufile.h>
+#else
+    #include "cufile_stub_types.h"
+#endif
```

Comment:
@grlee77 it looks like you added this change. Could you please share a bit more context?

Given the CTK has `cufile.h`, would think we could just use that `#include` directly

Is there some other issue that is coming up?